diff --git a/node_modules/onnxruntime-react-native/android/build.gradle b/node_modules/onnxruntime-react-native/android/build.gradle
index 41b4359..8cb2c8f 100644
--- a/node_modules/onnxruntime-react-native/android/build.gradle
+++ b/node_modules/onnxruntime-react-native/android/build.gradle
@@ -247,7 +247,10 @@ dependencies {
     extractLibs "com.microsoft.onnxruntime:onnxruntime-android:latest.integration@aar"
   }
 
-  if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
+  def __ortRnParts = REACT_NATIVE_VERSION.tokenize('.')
+  def __ortRnMajor = __ortRnParts[0].toInteger()
+  def __ortRnMinor = __ortRnParts.size() > 1 ? __ortRnParts[1].toInteger() : 0
+  if (__ortRnMajor == 0 && __ortRnMinor < 71) {
     extractLibs "com.facebook.fbjni:fbjni:+:headers"
     extractLibs "com.facebook.fbjni:fbjni:+"
   }
